cmd/goQuery/cmd/root.go

```diff
@@ -247,14 +247,23 @@ func entrypoint(cmd *cobra.Command, args []string) (err error) {
 
 	// check if arguments should be loaded from disk. The cmdLineParams are taken as
 	// the base for this to allow modification of single parameters
-	if viper.GetString(conf.StoredQuery) != "" {
-		argumentsJSON, err := os.ReadFile(filepath.Clean(argsLocation))
-		if err != nil {
-			return fmt.Errorf("failed to read query args from %s: %w", argsLocation, err)
+	argsLocation := viper.GetString(conf.StoredQuery)
+	if argsLocation != "" {
+		var argsReader io.Reader
```

Comment:
Code simplification: Initialize `argsReader` with `os.Stdin` and omit the `else` statement below:
```
var argsReader io.Reader = os.Stdin
if argsLocation != "-" {
...
}
```

Reply:
@els0r I've approved the PR, maybe you could still have a quick look at this tiny nitpick change?

Reply:
> @els0r I've approved the PR, maybe you could still have a quick look at this tiny nitpick change?

Never nesting. I hear ya